ci/conda/recipes/morpheus/meta.yaml

```diff
@@ -28,46 +28,16 @@ source:
   git_url: ../../../..
 
 outputs:
-  - name: morpheus-deps-build
-    build:
-      number: {{ GIT_DESCRIBE_NUMBER }}
-      string: cuda_{{ cuda_version }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-    requirements:
-      run:
-        - {{ compiler("c") }}
-        - {{ compiler("cxx") }}
-        - {{ compiler("cuda") }}
-        - ccache
-        - cmake=3.22
-        - cudatoolkit {{ cuda_version }}.*
-        - cudf {{ rapids_version }}
-        - cython >=0.29,<0.30
-        - librdkafka=1.6.1
-        - neo {{ minor_version }}.*
-        - ninja
-        - pip
-        - python {{ python }}
-        - rapidjson=1.1
-        - scikit-build>=0.12
-        - versioneer-518
-
-  - name: morpheus-deps-test
-    build:
-      number: {{ GIT_DESCRIBE_NUMBER }}
-      string: cuda_{{ cuda_version }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-    requirements:
-      run:
-        - cudatoolkit {{ cuda_version }}.*
-        - pytest
-        - pytest-cov
-        - python {{ python }}
 
   - name: morpheus
     build:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda_{{ cuda_version }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       script_env:
-        - CMAKE_CUDA_ARCHITECTURES=ALL
+        # Note: Setting a default value here will overwrite the value from the
+        # shell! Defaults should be set in the build script
+        - CMAKE_CUDA_ARCHITECTURES
+        - MORPHEUS_BUILD_PYTHON_STUBS
```

Comment:
@mdemoret-nv  Why were these added? They're cmake build args, not env variables.

Reply:
I see you've added the expectation of these being env variables as well, however the addition of these two lines is causing a build error.

Reply:
They are not all cmake build args but they do play a part in that. conda-build does not capture the environment from the calling shell by default. If you want to be able to tweak how conda-build is run in any way, you have to specify which environment variables that it should capture and specify them here.

This allows us to tweak how the the morpheus conda package is built from the calling shell. So if you set `CMAKE_CUDA_ARCHITECTURES=NATIVE ./ci/conda/recipes/run_conda_build.sh morpheus` you can speed up the build by only building for your GPU (helpful for testing). Or you can run: `MORPHEUS_BUILD_PYTHON_STUBS=OFF ./ci/conda/recipes/run_conda_build.sh morpheus` to avoid needing the NVIDIA driver during compilation (which we need in the Dockerfile)

Reply:
right, but it appears that you now _must_ specify those environment variables in the shell. Is this intentional?
```
Packaging morpheus-0.0.0.dev-cuda_11.5_py38_g2d493e0_71
INFO:conda_build.build:Packaging morpheus-0.0.0.dev-cuda_11.5_py38_g2d493e0_71
Traceback (most recent call last):
  File "/home/charris/miniconda3/bin/conda-mambabuild", line 10, in <module>
    sys.exit(main())
  File "/home/charris/miniconda3/lib/python3.9/site-packages/boa/cli/mambabuild.py", line 239, in main
    call_conda_build(action, config)
  File "/home/charris/miniconda3/lib/python3.9/site-packages/boa/cli/mambabuild.py", line 211, in call_conda_build
    result = api.build(
  File "/home/charris/miniconda3/lib/python3.9/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/home/charris/miniconda3/lib/python3.9/site-packages/conda_build/build.py", line 3088, in build_tree
    packages_from_this = build(metadata, stats,
  File "/home/charris/miniconda3/lib/python3.9/site-packages/conda_build/build.py", line 2371, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/home/charris/miniconda3/lib/python3.9/site-packages/conda_build/build.py", line 1602, in bundle_conda
    raise ValueError("env var '{}' specified in script_env, but is not set."
ValueError: env var 'MORPHEUS_BUILD_PYTHON_STUBS' specified in script_env, but is not set.
```

Reply:
Works now if I throw 'CMAKE_CUDA_ARCHITECTURES=ALL' in the Dockerfile at the morpheus conda build. 
```
#27 exporting to image
#27 exporting layers
#27 exporting layers 48.8s done
#27 writing image sha256:9773046de3ef35a3c8a5a359d9573b585d51be5f8786d21a181bf7d3529e213e done
#27 naming to nvcr.io/nvidia/morpheus/morpheus:v22.04.00-runtime done
#27 DONE 48.8s
```